media/stitcher/src/main/java/com/example/stitcher/CreateLiveConfig.java

```diff
@@ -0,0 +1,78 @@
+ * limitations under the License.
+ */
+
+package com.example.stitcher;
+
+// [START videostitcher_create_live_config]
+
+import com.google.cloud.video.stitcher.v1.AdTracking;
+import com.google.cloud.video.stitcher.v1.CreateLiveConfigRequest;
+import com.google.cloud.video.stitcher.v1.LiveConfig;
+import com.google.cloud.video.stitcher.v1.LiveConfig.StitchingPolicy;
+import com.google.cloud.video.stitcher.v1.LocationName;
+import com.google.cloud.video.stitcher.v1.SlateName;
+import com.google.cloud.video.stitcher.v1.VideoStitcherServiceClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateLiveConfig {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String liveConfigId = "my-live-config-id";
+    // Uri of the live stream to stitch; this URI must reference either an MPEG-DASH
+    // manifest (.mpd) file or an M3U playlist manifest (.m3u8) file.
+    String sourceUri = "https://storage.googleapis.com/my-bucket/main.mpd";
+    // See Single Inline Linear
+    // (https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/tags)
+    String adTagUri = "https://pubads.g.doubleclick.net/gampad/ads...";
+    String slateId = "my-slate-id";
+
+    createLiveConfig(projectId, location, liveConfigId, sourceUri, adTagUri, slateId);
+  }
+
+  public static void createLiveConfig(String projectId, String location, String liveConfigId,
+      String sourceUri, String adTagUri, String slateId)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (VideoStitcherServiceClient videoStitcherServiceClient =
+        VideoStitcherServiceClient.create()) {
+      CreateLiveConfigRequest createLiveConfigRequest =
+          CreateLiveConfigRequest.newBuilder()
+              .setParent(LocationName.of(projectId, location).toString())
+              .setLiveConfigId(liveConfigId)
+              .setLiveConfig(LiveConfig.newBuilder()
+                  .setSourceUri(sourceUri)
+                  .setAdTagUri(adTagUri)
+                  .setDefaultSlate(SlateName.format(projectId, location, slateId))
+                  .setAdTracking(AdTracking.SERVER)
+                  .setStitchingPolicy(StitchingPolicy.CUT_CURRENT).build())
+              .build();
+
+      LiveConfig response = videoStitcherServiceClient.createLiveConfigAsync(
+              createLiveConfigRequest)
+          .get(2, TimeUnit.MINUTES);
```

Comment:
2 minutes is used throughout the samples. Any thoughts on making that a constant in each file or `TestUtils`?

Reply:
Made this a constant in each sample file